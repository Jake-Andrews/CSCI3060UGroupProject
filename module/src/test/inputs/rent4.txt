login
USER
rent
91da4934
a943c2b1
14
yes
logout
exit