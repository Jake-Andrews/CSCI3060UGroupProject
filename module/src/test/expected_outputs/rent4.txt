Welcome, please login to use the program. 
To exit type exit, To start type login: 
Please enter a valid username: 
You have been logged in!

Please enter a command!
For a list of commands type 'help': Please enter the id of the unit you wish to rent: 
Please enter the id of the unit you wish to rent: 
Please enter a valid number of nights (integer value, minimum of 1, max of 14):
Are you sure you want to book a943c2b1 with a price per night of $400.0 , and a total cost of $5600?
Type 'yes' to accept or 'no' to decline: 
Processing your order!
Daily transaction file string: 05_USER_____________AA__a943c2b1__Toronto____________________3__400.0___0
Your order has been completed!

Please enter a command!
For a list of commands type 'help': Daily transaction file string: 00_USER_____________AA_______________________________0__0.0_____0

Printing Daily Transactions:05_USER_____________AA__a943c2b1__Toronto____________________3__400.0___0
00_USER_____________AA_______________________________0__0.0_____0
You have been successfully logged out!
Welcome, please login to use the program. 
To exit type exit, To start type login: Thank you for using our program!